CHANGELOG.md

```diff
@@ -3195,543 +3194,6 @@ This section is only relevant to contributors.
 
 - remove version from titlebar
 
-## [0.999.1] - 2020-02-02
```

Comment:
Why do we have to delete the old entries?

And then why do we still keep the links to them at the very bottom of the file?

Reply:
Why should we keep 5 year old changes in the changelog? But I'm not opinionated on the removal, so if there are reasons to keep them, fine for me

But yes if removed the links should be removed too